<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginWindow</class>
 <widget class="QWidget" name="LoginWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>646</width>
    <height>418</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LoginWindow</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="2">
    <widget class="QLineEdit" name="lineEdit_password"/>
   </item>
   <item row="4" column="2">
    <widget class="QPushButton" name="pushButton_login">
     <property name="text">
      <string>Вход</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="3">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Тип пользователя</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Пароль</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="2">
    <widget class="QLineEdit" name="lineEdit_username"/>
   </item>
   <item row="2" column="4">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="3">
    <widget class="QComboBox" name="comboBox"/>
   </item>
   <item row="4" column="3">
    <widget class="QPushButton" name="pushButton_registration">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Регистрация</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Логин</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
